packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -314,4 +314,35 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('getBlockStats', () => {
+    it('should get blockchain stats and return  all values', async () => {
```

Comment:
double spaces here..
overall the code gonna take note on alignment and space formatting